library/sonarqube

```diff
@@ -24,16 +24,21 @@ Tags: 8.9.10-datacenter-search, 8.9-datacenter-search, 8-datacenter-search, lts-
 Directory: 8/datacenter/search
 
 Tags: 9.7.1-community, 9.7-community, 9-community, community, latest
+Architectures: amd64, arm64v8
```

Comment:
should we support all ARM64 stuff ? 
- arm32v6
- arm32v7
- arm64v8

As we should focus only on M1 but knowing that everything is linked together ?

Reply:
In my opinion, we should just support `arm64v8` because our MMF is targeting Apple M1 only